### DISCUSSION TITLE
Latitude and longitude for calcHorizontalCoordinates() precision

### DISCUSSION THREAD

**Discussion by Macintosh-Fan** [jpb10/SolarCalculator#15] [Q&A] [answered]
Of what decimal precision works for the `void calcHorizontalCoordinates(unsigned long utc, double latitude, double longitude, double &azimuth, double &elevation)` function?

**Comment by jpb10** [comment, marked as answer]
For the input values of latitude and longitude, use 2 to 4 decimals.

```
double latitude = 45.55;     // Observer's latitude 
double longitude = -73.633;  // Observer's longitude
```

**Reply by Macintosh-Fan** [reply]
Great, thank you! Would 5 decimals still work, or does it only work reliably for up to 4?

**Reply by Macintosh-Fan** [reply]
Cool, thanks!

**Comment by jpb10** [comment]
Yes, you can use more decimals and it won't change the results.
